code/modules/arcade/prize_datums.dm

```diff
@@ -318,6 +318,12 @@ var/global/datum/prizes/global_prizes = new
 	typepath = /obj/item/twohanded/toy/chainsaw
 	cost = 200
 
+/datum/prize_item/headpat
+	name = "Gloves of Headpats"
+	desc = "Gloves that let you heatpat fast."
```

Comment:
"Gloves that fill you with an irresistable urge to give headpats."

Most of the arcade prizes have something witty about them